prime-router/metadata/hl7_mapping/common/datatype/xtn-telecom.yml

```diff
@@ -52,11 +52,23 @@ elements:
     value: [ '%resource.value.getPhoneNumberAreaCode()' ]
     hl7Spec: [ '%{hl7TelecomPath}-6' ]
 
+  # RADx MARS format wnats this value to be <deidentified>, but Hapi library does not allow that so setting it to empty
```

Comment:
wants*

Reply:
I'm still not sure what needs to be done to allow that. There's a separate ticket to investigate that. Should I add TODO and reference that ticket?